core-rust/engine-state-api-server/src/engine_state_api/conversions/common.rs

```diff
@@ -39,7 +39,7 @@ pub fn to_api_ledger_header_summary(
             transaction_tree_hash: to_api_transaction_tree_hash(&hashes.transaction_root),
             receipt_tree_hash: to_api_receipt_tree_hash(&hashes.receipt_root),
         }),
-        proposer_timestamp: Box::new(to_api_instant_from_safe_timestamp(
+        proposer_timestamp: Box::new(to_api_consensus_instant(
```

Comment:
Can we tweak this to say `_from_timestamp_ms` to allow us to be able to verify each line for correctness?